modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -36,6 +36,9 @@ and upstream registries.
 |`Storagetype`: Details for configuring registry storage, for example S3 bucket
 coordinates. Normally configured by default.
 
+|`readOnly`
```

Comment:
It's better to keep consistent with other parameters, start with upper case letter.